src/content/pages/FileUpload/ParameterModal.tsx

```diff
@@ -0,0 +1,85 @@
+
+import React, { useState } from 'react'
+import {
+  Box,
+  IconButton,
+  Modal,
+  Tab,
+  Tabs,
+  Typography,
+  TableContainer,
+  Table,
+  TableHead,
+  TableBody,
+  TableRow,
+  TableCell
+} from '@mui/material'
+import { modalStyle } from './modalStyle'
+import CancelIcon from '@mui/icons-material/Cancel'
+
+import parameters from '../../../../../sharedData/parameterBuckets.json'
+
+interface TabPanelProps {
+  children?: React.ReactNode
+  index: number
+  value: number
+}
+
+function TabPanel(props: TabPanelProps) {
+  const { children, value, index} = props
+
+  return (
+    <Box sx={{maxHeight:'50%'}}>
+      {value === index && (
+        <Box sx={{ p: 3, maxHeight:'50%' }}>
+          {children}
+        </Box>
+      )}
+    </Box>
+  )
+}
+
+export default function ParameterModal({ openParameterModal, setOpenParameterModal }) {
+  const [tab, setTab] = useState(0)
+  return (
+    <Modal open={openParameterModal} onClose={() => setOpenParameterModal(false)}>
+      <Box sx={{ minWidth: '50%', height: '50%', ...modalStyle }}>
```

Comment:
```suggestion
      <Box sx={{ minWidth: '50%', ...modalStyle }}>
```
The addition of the `height: '50%'` part caused some of the listed items to render outside of the modal

![image](https://user-images.githubusercontent.com/101340211/195315804-f8217b1b-2b66-4db4-9dbd-bbc482a87366.png)

I tested removing this part, and that fixed the issue

Reply:
Sweet ill approve this